### DISCUSSION TITLE
How could I interrupt the current playing dialogue?

### DISCUSSION THREAD

**Discussion by HakkerBarry** [nathanhoad/godot_dialogue_manager#480] [Q&A]
Hi folks,

I want my dialogue to be interrupted by enemy’s attacks. This event is not driven by the conversation itself, but is triggered by an external function. Since I disabled my input during playing a dialogue, is there any way I can interrupt the playing dialogue immediately and enable my input again? 

Thanks!

**Comment by comradecid** [comment]
this has been driving me bloody bonkers. we have the ability to start a balloon, but not end it gracefully. i tried hot-wiring it to force it to an 'END' state through various methods in an attempt to get it to dismiss gracefully, but it doesn't work.

do let me know if you've found a way to dismiss the balloon without going scorched-earth and doing a queue_free on it...
